### PR TITLE
Updating the Raspberry Pi directions

### PR DESCRIPTION
**Pull request by curbtripper**
After creating your pull request, tick these boxes if they are applicable to you.

- [ ] I have tested my changes against the `review` branch (the latest developmental version), and this pull request is targeting that branch as a base
- [ ] I have tested my changes on Python 3.5/3.6

----

### Description

raspbian already comes with sudo, and all the seperate apt-get commands weren't really necessary. also, the repositories don't work on raspbian and "make -j4" should be used on newer pi's (like the 2B, 3B, and 3B+, but not the zero or zero w)

### Related issues (if applicable)

running through the steps as is appears to install a version of pip and then proceed to replace it with a newer version, is this step really necessary?

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# axisps-musicbot
+
+Just ignore this fork because it's really outdated
```
Comment (MattBSG) [inline readme.md]: Please revert or remove the readme.md changes as it doesn’t pertain to our repository.

**Review by MattBSG — CHANGES_REQUESTED** [review]
Looks good other than the one change I commented on, just unable to test as I do not have a pi.

File: _docs/installing/raspbian.md [hunk 2]
```diff
@@ -10,28 +10,17 @@ Installing MusicBot on Raspbian for use with a Raspberry Pi 2 or 3B is a **long-
...
-sudo apt-get install build-essential libncursesw5-dev libgdbm-dev libc6-dev
-sudo apt-get install zlib1g-dev libsqlite3-dev tk-dev
-sudo apt-get install libssl-dev openssl
-sudo apt-get install build-essential unzip -y
-sudo apt-get install software-properties-common -y
-sudo add-apt-repository ppa:deadsnakes -y
-sudo add-apt-repository ppa:mc3man/trusty-media -y
-sudo add-apt-repository ppa:chris-lea/libsodium -y
-sudo apt-get install libopus-dev libffi-dev libsodium-dev
+sudo apt-get install git build-essential libncursesw5-dev libgdbm-dev libc6-dev zlib1g-dev libsqlite3-dev tk-dev libssl-dev openssl unzip software-properties-common libopus-dev libffi-dev libsodium-dev -y
```
Comment (MattBSG) [inline _docs/installing/raspbian.md:15]: It is split up to group certain types of dependencies and make it cleaner. You also removed the `apt-add-repository`s which some packages come from

Comment (curbtripper) [reply _docs/installing/raspbian.md:15]: The repositories only report this error when trying to run them with Raspbian:

`Traceback (most recent call last):                                              
  File "/usr/bin/add-apt-repository", line 95, in <module>                      
    sp = SoftwareProperties(options=options)                                    
  File "/usr/lib/python3/dist-packages/softwareproperties/SoftwareProperties.py"
, line 109, in __init__                                                         
    self.reload_sourceslist()                                                   
  File "/usr/lib/python3/dist-packages/softwareproperties/SoftwareProperties.py"
, line 599, in reload_sourceslist                                               
    self.distro.get_sources(self.sourceslist)                                   
  File "/usr/lib/python3/dist-packages/aptsources/distro.py", line 89, in get_so
urces                                                                           
    (self.id, self.codename))                                                   
aptsources.distro.NoDistroTemplateException: Error: could not find a distributio
n template for Raspbian/stretch`

[asciinema example of said errors](https://asciinema.org/a/175275)

File: readme.md (hunk 1 above)
Comment (curbtripper) [reply readme.md]: removed

**Review by MattBSG — APPROVED** [review]

**Comment by jayktaylor** [comment]
#1671 appears to have superseded this.

**Pull request closed without merging**